lib/jenkins_pipeline/app.rb

```diff
@@ -2,6 +2,7 @@ module JenkinsPipeline
   class App < Sinatra::Base
 
     set :public_folder, './public'
+    set :bind, '0.0.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.